cms/djangoapps/contentstore/utils.py

```diff
@@ -455,3 +457,16 @@ def get_visibility_partition_info(xblock):
         "has_selected_groups": has_selected_groups,
         "selected_verified_partition_id": selected_verified_partition_id,
     }
+
+
+def is_self_paced(course):
```

Comment:
I don't think this is worthwhile to abstract out into its own utility. Also, `self_paced` should always return a boolean: https://github.com/edx/edx-platform/blob/master/common/lib/xmodule/xmodule/course_module.py#L753-L763.

If it's not set, it'll return `False`